### PR TITLE
JSON Manager: Adding wb-swap and patches

### PR DESCRIPTION
**Pull request by Garneauma**
Adding new features to JSON Manager:
- "wb-swap": replace a value from defined path with a value from another defined path (usually a reference JSON)
- "patches": allows to define a new base path while still having access to the original path (URL)
- Possibility to define multiple URLs for one dataset

This PR is dependent on @duboisp's work on JSON Manager merge feature.

### CODE REVIEW

File: src/plugins/wb-jsonmanager/jsonmanager-fr.hbs [hunk 1]
```diff
@@ -116,6 +118,334 @@
...
+				"french": "Français"
+			}
+		},
+		"province": {
+			"en": {
+				"alberta": "Alberta",
+				"british_columbia": "British Columbia",
+				"manitoba": "Manitoba",
+				"new_brunswick": "New Brunswick",
+				"newfoundland_labrador": "Newfoundland & Labrador",
```
Comment (duboisp) [inline src/plugins/wb-jsonmanager/jsonmanager-fr.hbs:292]:
```suggestion
				"newfoundland_labrador": "Newfoundland &amp; Labrador",
```

File: src/plugins/wb-jsonmanager/jsonmanager-fr.hbs [hunk 2]
```diff
@@ -116,6 +118,334 @@
...
```
Comment (duboisp) [inline src/plugins/wb-jsonmanager/jsonmanager-fr.hbs:409]:
```suggestion
				"newfoundland_labrador": "Newfoundland &amp; Labrador",
```

File: site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs [hunk 3]
```diff
@@ -232,402 +243,405 @@
...
-								{
-									"selector": "dd",
-									"path": "desc"
-								}] } ] &hellip;}</code> </td>
-<td>Used for combined selectors e.g. description lists. Same configuration as for outer <code>extractor</code> option, it is an array of objects containing html selectors with their specific paths.
-
-</tr>
-</tbody>
-</table>
+	<h2>Configuration options</h2>
```
Comment (duboisp) [inline site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs:271]: Need to add a blurb for the new option "concat"

File: site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs [hunk 4]
```diff
@@ -232,402 +243,405 @@
...
+				<td>You can follow the url or the dataset name by a <a href="https://tools.ietf.org/html/rfc6901">JSON Pointer (RFC6901)</a>.</td>
+				<td>
+					<dl>
+						<dt><code>data-wb-jsonmanager='{ "url": "location/of/json/file.json#/" }'</code></dt>
+						<dd>The url are a json file</dd>
+						<dt><code>data-wb-jsonmanager='{ "url": "[datasetName]#/" }'</code></dt>
+						<dd>The url is a reference to a dataset managed by a json-manager defined in the same page</dd>
+					</dl>
+				</td>
+			</tr>
```
Comment (duboisp) [inline site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs:294]: Complement the description to say that we can nom fetch multiple JSON file + an additional code sample in the description list.

**Review by duboisp — CHANGES_REQUESTED** [review]
I did a full code review + local testing, everything work as expected. This PR is going to be a minor change.

* [x] Add support for patches operation inside patches, see the propose changes and demoed here: http://universallabs.org/wet/labs/acr/unmin/demos/acr/demo-wcag21/acr-wcag21-en.html
* [x] Add a working example where we show a patches inside patches like for the ACR report.
* [x] Documents mulitple urls + contact configuration options
* [x] Fix a few HTML escaping

**Comment by duboisp** [comment]
Side note: This PR is required for the revised accessibility conformance report methodology as discussed in #9271

File: site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs (hunk 3 above)
Comment (Garneauma) [reply site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs:271]: Following our discussion, the "concat" option has been removed. Arrays will be concatenated by default and objects extended.

File: site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs [hunk 5]
```diff
@@ -232,402 +243,407 @@
...
+				<th>Description</th>
+				<th>How to configure</th>
+				<th>Values</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td><code>url</code></td>
+				<td><strong>Required</strong>. Define the url or the dataset name to use. When used in a template mode, the URL should point to an array object.</td>
+				<td>You can follow the url or the dataset name by a <a href="https://tools.ietf.org/html/rfc6901">JSON Pointer (RFC6901)</a>. The value can be either a string or an array of strings. If all references are arrays, the arrays will be concatenated into one array. If all references are objects, the objects will be extended into a single object. If there are references of multiple types, the plugin will throw an error.</td>
```
Comment (duboisp) [inline site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs:285]: L'ajout de contenu ici doit être répliqué dans la page française.

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Pre-approved after the requested changes is completed and after a final review is completed.

This PR is a **Minor change**, add 3 functionality to JSON manager
* Load multiple URLs to build the data set
* Add patch `wb-swap`
* Add patch `patches`

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally, work as expected.

**Pull request merged**